### PR TITLE
adding basic post creation

### PR DESCRIPTION
**Pull request by agugneja**
Created but isn't working. My VSCode is insisting some modules aren't downloaded, wanted to see if it's that or a mistake in my coding. Anyone see an error?

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -24,6 +26,25 @@ def encyclopedia():
 def sightings():
     return render_template('sightings.html')
 
+def create_post():
```
Comment (GingerWookiee) [inline app.py:29]: Shouldn't the CRUD functions be in a repositories folder, not in app.py?

File: templates/sightings.html [hunk 2]
```diff
@@ -4,6 +4,20 @@
 
 {% block body %}
 
-<!-- Posts of various monsters containing an image, header, and caption -->
+<h1>Sightings</h1>
+<form>
```
Comment (GingerWookiee) [inline templates/sightings.html:8]: for your form to work, you need an action="" with a /something that links to a @app.get("/something") or @app.post("/something")

**Review by GingerWookiee — CHANGES_REQUESTED** [review]

**Review by awramos — APPROVED** [review]
lgtm

**Review by awramos — APPROVED** [review]
lgtm! just need to look at whats conflicting in app.py

**Pull request merged**